src/utils/api.ts

```diff
@@ -34,3 +34,57 @@ export const searchOrders = async (
   const response = await axios.request(request);
   return response.data;
 };
+
+export const getCountries = async (
+  authData: TAuthData,
+  data: { pageNo: number; pageSize: number }
+): Promise<unknown> => {
+  const { server, token } = authData;
+  const { pageNo, pageSize } = data;
+  const request: AxiosRequestConfig = {
+    url: `${server}/api/v1/countries?page_no=${pageNo}&page_size=${pageSize}`,
```

Comment:
Consider creating an Axios instance with a predefined base URL and using it throughout the project instead of directly using Axios. This will improve maintainability, enable easier configuration (e.g., adding headers, interceptors), and ensure consistency across API calls.